Chapters/02-statistics.Rmd

```diff
@@ -831,22 +831,24 @@ This is a very simple method that allows us to get to the model closest to the t
     d_j = k_j - \ell_j
     (\#eq:AICNormal),
 \end{equation}
-where $k_j$ is the number of all estimated parameters in the model $j$ (this typically also includes scale parameters, when dealing with Maximum Likelihood Estimates). When studying the properties of \@ref(eq:AICNormal), @Akaike1974 suggested to multiply both parts of the right hand side by 2, so that there is a connection between the proposed criterion and the well known likelihood-ratio test [@WikipediaLikelihoodRatioTest2020], and he proposed the following "An Information Criterion":
+where $k_j$ is the number of parameters in model $j$ (this typically includes scale parameters when dealing with Maximum Likelihood Estimates). @Akaike1974 suggests "An Information Criterion" which multiplies both parts of the right-hand side of \@ref(eq:AICNormal) by 2 so that there is a correspondence between the criterion and the well-known likelihood ratio test [@WikipediaLikelihoodRatioTest2020]:
```

Comment:
I disagree with the removal of "all estimated" - this is discussed in the next section. If we don't estimate something, then i shouldn't count. I agree with the rest.

Reply:
Fair point!